test/unit/document_test.rb

```diff
@@ -141,6 +141,32 @@ def test_router__inheritance
       assert_equal router2, page_2_3_2.router
     end
 
+    def test_reload
+      kitty_class = Class.new(AePageObjects::Document)
+
+      stub_current_window
+
+      kitty_page = kitty_class.new
+      capybara_stub.session.driver.expects(:execute_script) do |script|
+        puts script
```

Comment:
Do we want this `puts`?

Reply:
Good catch, I was supposed to put assertions here about the contents of the script but missed it because the test passed!  Now fixed.